netutils/lib_mapper.py

```diff
@@ -247,6 +252,7 @@
     "pluribus": "community.network.netvisor",
     "ruckus_icx": "community.network.icx",
     "vyos": "vyos.vyos.vyos",
+    "aruba_aoscx": "arubanetworks.aoscx",
```

Comment:
Sorry, I may have caused some confusion. I thought you were asking about adding `aruba_aoscx` to libraries where they existed, but this does not exist in pyntc as an example. This mapper is intended to map between what a given library calls it.

Reply:
All the mappings should exist. 
Ansible: https://galaxy.ansible.com/arubanetworks/aoscx
Napalm: https://github.com/napalm-automation-community/napalm-aruba-cx
Scrapli: https://github.com/scrapli/scrapli_community/tree/main/scrapli_community/aruba/aoscx